index.bs

```diff
@@ -1817,9 +1914,10 @@ Build a composed graph up to a given output operand into a computational graph a
         1. Set |graph|.{{MLGraph/[[outputDescriptors]]}}[|name|] to |operand|.{{MLOperand/[[descriptor]]}}.
     1. Set [=this=].{{MLGraphBuilder/[[hasBuilt]]}} to true.
     1. Let |promise| be [=a new promise=] in |realm|.
-    1. Run the following steps [=in parallel=]:
+    1. Enqueue the following steps to |graph|.{{MLGraph/[[context]]}}.{{MLContext/[[timeline]]}}:
         1. Run these steps, but [=/abort when=] |graph|.{{MLGraph/[[context]]}} [=MLContext/is lost=]:
             1. Let |graphImpl| be the result of converting [=this=]'s [=MLGraphBuilder/graph=] with |operands|, |operators|, |inputs|, and |outputs|'s [=map/values=] into an [=implementation-defined=] format which can be interpreted by the underlying platform.
+                1. During this step, for each |operand| in |operands|, if |operand| was created using {{MLGraphBuilder/constant(tensor)|constant()}} with a given {{MLConstantTensor}} |tensor|, the {{MLTensor/[[data]]}} of |tensor| may be accessed on the {{MLContext/[[timeline]]}}.
```

Comment:
Note that "may" here in normative has the special RFC 2119 meaning, i.e. that it's a optional rather than a requirement for implementations, which is probably not what is intended.

Also, since these steps are enqueued to the context's timeline, it's implicit that all of this work is running on that timeline, so I don't think this step is necessary.

Reply:
Removed.